guides/common/modules/proc_promoting-a-content-view.adoc

```diff
@@ -29,7 +29,7 @@ Now the repository for the Content View appears in all environments.
 
 [id="cli-promoting-a-content-view_{context}"]
 .CLI procedure
-* Promote the Content View using the `hammer content-view version promote` each time:
+. Promote the Content View using the `hammer content-view version promote` each time:
```

Comment:
```suggestion
* Promote the Content View using Hammer for each lifecycle environment:
```
I think it's better to stick with unnumbered list, because the list items denote alternative steps, not a sequence of steps.